### PR TITLE
Added a way to get install's TLS version.

### PR DESCRIPTION
**Pull request by Much2tall**
Added function to get the current install's TLS version and added a call to that function to the System Status page.

### CODE REVIEW

**Comment by Much2tall** [comment]
Needed for updates to PayPal and Stripe that catch old TLS versions.

Closes: https://github.com/wpninjas/ninja-forms/issues/2571

File: includes/Helper.php [hunk 1]
```diff
@@ -239,6 +239,45 @@ public static function maybe_unserialize( $original )
...
+     * 
+     * Since 3.0
+     * 
+     * @return float OR false
+     */
+    public static function get_tls()
+    {
+        $php_ver = phpversion();
+        // If we have a php version of 5.6 or higher...
+        if( '5.6.0' > $php_ver ) return false;
```
Comment (kjohnson) [inline includes/Helper.php]: Shouldn't we be using PHP's [version_compare](http://php.net/manual/en/function.version-compare.php) function?

Comment (Much2tall) [inline includes/Helper.php]: Good catch.

Comment (Much2tall) [inline includes/Helper.php]: Updated

**Pull request merged**